doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,494 @@
+================================================
+Tutorial: Linear algebra on n-dimensional arrays
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+**Prerequisites**
+
+Before reading this tutorial, you should know a bit of Python. If you
+would like to refresh your memory, take a look at the `Python
+tutorial <https://docs.python.org/tutorial/>`__.
```

Comment:
Could be changed to `tutorial-index`, although that is a bit generic, it is the official intersphinx link to the page. Maybe too generic to be useful: if we ever create a `tutorial-index` heading I don't know what would take precedence

Reply:
I used the suggestion from above and changed it to `<python:tutorial/index>`